website/content/en/status/report-2024-10-2024-12/clusteradm.adoc

```diff
@@ -0,0 +1,81 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronize its distributed work and communications.
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Cluster software refresh.
+* Moving more cluster services to Chicago.
+* Supporting the Grimoirelab dashboard effort.
+
+==== Cluster software refresh
+
+Except for the package builders and developer-facing ("dogfood") machines, the FreeBSD cluster mostly tracks stable/X branches.
+
+At the time of this writing, there are 131 physical machines in the cluster.
+We have 54 machines on current, 61 on stable/14 and 14 on stable/13.
+Work continues to upgrade the remaining stable/13 machines to stable/14.
+The stable/12 machines have been slated for decommissioning for a while, they do not run production workloads.
```

Comment:
Optional grammar nit: You are separating two independent clauses with the comma.

How about:

The stable/12 machines have been slated for decommissioning for a while.
They do not run production workloads.

or 

The stable/12 machines have been slated for decommissioning for a while; they do not run production workloads.